### PR TITLE
add more to expanded definition

### PR DESCRIPTION
**Pull request by fluffy**
Fixes https://github.com/core-wg/senml-spec/issues/20

### CODE REVIEW

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -351,11 +351,20 @@ Representing the statistical characteristics of measurements, such as accuracy,
 can be very complex. Future specification may add new attributes to provide
 better information about the statistical properties of the measurement.
 
+## Expanded Records
+
+Sometimes it is useful to be able to refer to a defined normalized format for
+SenML records is useful for big data applications and intermediate forms.
```
Comment (akeranen) [inline draft-ietf-core-senml.md]: typo "records is useful"

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -351,11 +351,20 @@ Representing the statistical characteristics of measurements, such as accuracy,
...
-values and has no relative times, but the the base values of the SenML Pack
-(if any) are applied to the Record. That is, name and base name are
-concatenated, base time is added to the time of the Record, if Record did 
-not contain Unit the Base Unit is applied to the record, etc.
+values and has no relative times, but the the base values of the SenML Pack (if
+any) are applied to the Record. That is, name and base name are concatenated,
+base time is added to the time of the Record, if Record did not contain Unit the
+Base Unit is applied to the record, etc. In addition the records need to be in
+chronological order and for XML and JSON formats, each
+record is on one line with no extra whitespace.
```
Comment (akeranen) [inline draft-ietf-core-senml.md]: Is the last constraint (one line with no extra ws) useful beyond pretty-printing?

Comment (cabo) [inline draft-ietf-core-senml.md]: Hi Ari,

I think the word we are looking for is "resolved".  

Grüße aus der Wartburg, Carsten

**Comment by fluffy** [comment]
Updated to add in Air and Carsten sugestions

**Comment by akeranen** [comment]
Looks good to me

**Pull request merged**